### ISSUE TITLE
Error when requiring evil-lisp-state

### ISSUE THREAD

**Issue by rcoedo** [syl20bnr/evil-lisp-state#20]
I'm getting the following error when requiring this package:

```
Debugger entered--Lisp error: (void-variable evil-lisp-state-major-mode-map)
  (define-key evil-lisp-state-major-mode-map "%" (evil-lisp-state-enter-command evil-jump-item))
  (if evil-lisp-state-global (define-key evil-lisp-state-map "%" (evil-lisp-state-enter-command evil-jump-item)) (define-key evil-lisp-state-major-mode-map "%" (evil-lisp-state-enter-command evil-jump-item)))
  (progn (if evil-lisp-state-global (define-key evil-lisp-state-map "%" (evil-lisp-state-enter-command evil-jump-item)) (define-key evil-lisp-state-major-mode-map "%" (evil-lisp-state-enter-command evil-jump-item))))
  eval((progn (if evil-lisp-state-global (define-key evil-lisp-state-map "%" (evil-lisp-state-enter-command evil-jump-item)) (define-key evil-lisp-state-major-mode-map "%" (evil-lisp-state-enter-command evil-jump-item)))))
  (let ((key (car x)) (cmd (cdr x))) (eval (list (quote progn) (list (quote if) (quote evil-lisp-state-global) (list (quote define-key) (quote evil-lisp-state-map) (kbd key) (list (quote evil-lisp-state-enter-command) cmd)) (list (quote define-key) (quote evil-lisp-state-major-mode-map) (kbd key) (list (quote evil-lisp-state-enter-command) cmd))))))
  (while --dolist-tail-- (setq x (car --dolist-tail--)) (let ((key (car x)) (cmd (cdr x))) (eval (list (quote progn) (list (quote if) (quote evil-lisp-state-global) (list (quote define-key) (quote evil-lisp-state-map) (kbd key) (list (quote evil-lisp-state-enter-command) cmd)) (list (quote define-key) (quote evil-lisp-state-major-mode-map) (kbd key) (list (quote evil-lisp-state-enter-command) cmd)))))) (setq --dolist-tail-- (cdr --dolist-tail--)))
  (let ((--dolist-tail-- evil-lisp-state-commands) x) (while --dolist-tail-- (setq x (car --dolist-tail--)) (let ((key (car x)) (cmd (cdr x))) (eval (list (quote progn) (list (quote if) (quote evil-lisp-state-global) (list (quote define-key) (quote evil-lisp-state-map) (kbd key) (list ... cmd)) (list (quote define-key) (quote evil-lisp-state-major-mode-map) (kbd key) (list ... cmd)))))) (setq --dolist-tail-- (cdr --dolist-tail--))))
  eval-buffer
```

Any suggestion?

**Comment by Benaiah** [comment]
I'm getting this too since updating my packages.

**Comment by rcoedo** [comment]
I've made a pull request to fix this one month ago, but I got no response yet.

You can use my fork until the fix is merged: `https://github.com/rcoedo/evil-lisp-state`

If you use cask you can point to my fork like this:
`(depends-on "evil-lisp-state" :git "https://github.com/rcoedo/evil-lisp-state")`

I hope this helps!

**Comment by syl20bnr** [comment]
OMG sorry I missed this one. I will merge it soon.

**Comment by syl20bnr** [comment]
Thank you for the fix :+1: 
Will be fixed when 8.2 hit MELPA.
